decidim-conferences/app/cells/decidim/conferences/conference_g_cell.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Conferences
+    # This cell renders the Grid (:g) conference card
+    # for an given instance of an Assembly
+    class ConferenceGCell < Decidim::CardGCell
+      private
+
+      def resource_path
+        Decidim::Conferences::Engine.routes.url_helpers.conference_path(model)
+      end
+
+      def resource_image_path
```

Comment:
This is the same method as defined in AssemblyGCell

Reply:
Replied [there](https://github.com/decidim/decidim/pull/10219#discussion_r1086648699)